src/pages/components/tiles.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Tiles
+description: >
+  Tiles are Radio buttons styled with icons or images.
+package: '@zendeskgarden/react-forms'
+components:
+  - forms/src/elements/tiles/components/Tile.tsx
+  - forms/src/elements/tiles/Tiles.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>
+        <Markdown>
+          Like [Radios](/components/radio), Tiles are for choices that can't occur at the same time
+        </Markdown>
+      </li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          To choose more than one option at once, use a [Checkbox](/components/checkbox) instead
+        </Markdown>
+      </li>
+      <li>
+        <Markdown>
+          To select from a long list of options, use a [Select](/components/select)
+        </Markdown>
+      </li>
+      <li>
+        <Markdown>To define an action, use a [Button](/components/button) instead</Markdown>
+      </li>
+      <li>
+        <Markdown>For navigation, use an [Anchor](/components/anchor) instead</Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+Tile content is centered by default.
```

Comment:
If this is the default, how else can the content be displayed? Feels like there is a key missing example based on `isCentered={false}`.

Reply:
Maybe we follow suit with other component pages and give this: `The typical usage of a Tiles component.` and leave the layout commentary to the `Layout` example. I'll move in that direction for now.